### PR TITLE
policy in wasm

### PR DESCRIPTION
**Pull request by nathanjzhao**

### CODE REVIEW

File: examples/scenes/humanoid.xml [hunk 1]
```diff
@@ -65,9 +67,9 @@
       </default>
 
       <!-- joints -->
-      <joint type="hinge" damping=".4" stiffness="2" armature=".01" limited="true" solimplimit="0 .99 .01"/>
+      <joint type="hinge" damping=".2" stiffness="1" armature=".01" limited="true" solimplimit="0 .99 .01"/>
       <default class="joint_big">
-        <joint damping="10" stiffness="20"/>
+        <joint damping="5" stiffness="10"/>
```
Comment (budzianowski) [inline examples/scenes/humanoid.xml:72]: Did you retrain the original policy with these values?

Comment (nathanjzhao) [reply examples/scenes/humanoid.xml:72]: nah

Comment (budzianowski) [reply examples/scenes/humanoid.xml:72]: Does the policy work then without changing these parameters?

Comment (nathanjzhao) [reply examples/scenes/humanoid.xml:72]: yea

Comment (budzianowski) [reply examples/scenes/humanoid.xml:72]: So why changing them?

Comment (nathanjzhao) [reply examples/scenes/humanoid.xml:72]: uh forgot to change back woops

**Pull request merged**